### PR TITLE
chore(typescript): driverProviders converted to typescript

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by juliemr** [comment]
Travis errors are legitimate.

File: lib/globals.d.ts [hunk 1]
```diff
@@ -1 +1,11 @@
 declare var browser: any;
+
+declare class WebDriver {
```
Comment (juliemr) [inline lib/globals.d.ts]: I'm in favor of everything that's required from WebDriver being put in a namespace.

File: typings.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
 {
   "ambientDependencies": {
     "chalk": "registry:dt/chalk#0.4.0+20160317120654",
+    "form-data": "registry:dt/form-data#0.0.0+20160316155526",
```
Comment (juliemr) [inline typings.json]: What are the form-data typings for?

Comment (cnishina) [inline typings.json]: When installing `request` it wanted to install `form-data`. It is on my radar to remove `request`...

File: lib/globals.d.ts (hunk 1 above)
Comment (cnishina) [inline lib/globals.d.ts]: Yup, after seeing your pr... I am going to change mine to match.

**Comment by cnishina** [comment]
#1203

**Comment by juliemr** [comment]
Looks good! Merge it.

**Pull request merged**